### PR TITLE
Adjust minimum triangle movement speed to avoid "static" triangles in logo

### LINKED ISSUE #14584: In main menu on osu! logo some little triangle(s) moves too slow
So, the problem is already described in title. I tried to restart the client but this triangle (sometimes one, sometimes two) spawns anyway at different places.
Video:

https://user-images.githubusercontent.com/47123924/131412211-9d979618-015b-474a-829c-84ab39a65693.mp4

### PR DESCRIPTION
**Pull request by peppy**
Closes #14584.

### CODE REVIEW

File: osu.Game/Graphics/Backgrounds/Triangles.cs [hunk 1]
```diff
@@ -153,7 +153,7 @@ protected override void Update()
                 TriangleParticle newParticle = parts[i];
 
                 // Scale moved distance by the size of the triangle. Smaller triangles should move more slowly.
```
Comment (frenzibyte) [inline osu.Game/Graphics/Backgrounds/Triangles.cs:155]: Seems like this comment is no longer valid (the second portion specifically). Not sure how to reword it though.

Comment (peppy) [reply osu.Game/Graphics/Backgrounds/Triangles.cs:155]: It's still valid? The only different is it's clamped slightly now.

Comment (frenzibyte) [reply osu.Game/Graphics/Backgrounds/Triangles.cs:155]: Hmm, got more unsure with that "more slowly", but yeah nevermind.

**Review by frenzibyte — APPROVED** [review]

**Review by bdach — APPROVED** [review]
manually tested by affixing the particles' scale to always be `0.1f`, definitely does look to be moving faster than previously

**Pull request merged**